src/commands/config-schema.json

```diff
@@ -99,25 +99,71 @@
         },
         "colors": {
             "type": "object",
-            "description": "Mapping from jj formatter labels to color names",
+            "description": "Mapping from jj formatter labels to colors",
+            "definitions": {
+                "colors": {
+                    "enum": [
+                        "black",
+                        "red",
+                        "green",
+                        "yellow",
+                        "blue",
+                        "magenta",
+                        "cyan",
+                        "white",
+                        "bright black",
+                        "bright red",
+                        "bright green",
+                        "bright yellow",
+                        "bright blue",
+                        "bright magenta",
+                        "bright cyan",
+                        "bright white"
+                    ]
+                },
+                "basicFormatterLabels": {
+                    "enum": [
```

Comment:
Getting a little elaborate here, but WDYT of including some basic label names in the schema? The way I specified them doesn't restrict you to these names but could provide a hint to schema-aware tooling that these are possible setting names.

Reply:
Makes sense.